store/copr/coprocessor.go

```diff
@@ -981,6 +981,10 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *tikv.R
 		logutil.Logger(bo.GetCtx()).Warn("other error",
 			zap.Uint64("txnStartTS", worker.req.StartTs),
 			zap.Uint64("regionID", task.region.GetID()),
+			zap.Uint64("bucketsVer in request", task.bucketsVer),
+			zap.Uint64("bucketsVer in response", resp.pbResp.GetLatestBucketsVersion()),
+			zap.Int("number of ranges", task.ranges.Len()),
+			zap.String("start and end keys in ranges", task.ranges.String()),
```

Comment:
I'm afraid it can be too long. I think log the first key and the last key of the ranges is enough, because it represents the range of the bucket.

Reply:
That's what I thought before. I might have to add another method for KvRange.